library/src/main/java/com/pddstudio/highlightjs/HighlightJsView.kt

```diff
@@ -39,6 +42,8 @@ class HighlightJsView : WebView, FileUtils.Callback {
     private var onLanguageChangedListener: OnLanguageChangedListener? = null
     private var onThemeChangedListener: OnThemeChangedListener? = null
     private var onContentChangedListener: OnContentChangedListener? = null
+    var onSelectionChange: SelectionCallback? = null
```

Comment:
Either:

- Rename `HighlightJsView.onSelectionChange` to `selectionCallback` (for consistency with the `SelectionCallback` type).

or:

- Change the type of `onSelectionChange` to `((String?) -> Unit)?` and delete `SelectionCallback`.

?

(My preference goes to making `onSelectionChange` a function type. And also to renaming it to `onSelectionChange*d*`.)